### PR TITLE
Update context file URL

### PR DESCRIPTION
**Pull request by clehner**
Address #21

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -323,7 +323,7 @@ <h3>RevocationList2021Status</h3>
 {
   "@context": [
     "https://www.w3.org/2018/credentials/v1",
-    "https://w3id.org/vc-status-list-2021/v1"
+    "https://w3id.org/vc/status-list/v1"
```
Comment (msporny) [inline index.html:326]: I think we're going to have to go back to date-stamped values here... see https://github.com/w3c-ccg/vc-status-list-2021/issues/3#issuecomment-1062428949

**Review by msporny — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (msporny) [reply index.html:326]: Alright, done in https://github.com/w3c-ccg/vc-status-list-2021/pull/24 -- let me know what you think, @clehner.

**Review by OR13 — CHANGES_REQUESTED** [review]
Fix context links for close PR.

**Comment by msporny** [comment]
> https://w3id.org/vc/status-list/v1

The correct (and only) URL is now: https://w3id.org/vc/status-list/2021/v1 ... but spec is waiting for PR #24 to be merged to make that correction.

@clehner -- we should probably close this PR, leaving it open is going to confuse people. This PR is now superseded by PR #24.

**Comment by OR13** [comment]
Closing, reopen if you disagree.

**Pull request closed without merging**